servers/rendering_server.cpp

```diff
@@ -3513,7 +3513,7 @@ void RenderingServer::init() {
 	GLOBAL_DEF(PropertyInfo(Variant::INT, "rendering/textures/decals/filter", PROPERTY_HINT_ENUM, "Nearest (Fast),Linear (Fast),Nearest Mipmap (Fast),Linear Mipmap (Fast),Nearest Mipmap Anisotropic (Average),Linear Mipmap Anisotropic (Average)"), DECAL_FILTER_LINEAR_MIPMAPS);
 	GLOBAL_DEF(PropertyInfo(Variant::INT, "rendering/textures/light_projectors/filter", PROPERTY_HINT_ENUM, "Nearest (Fast),Linear (Fast),Nearest Mipmap (Fast),Linear Mipmap (Fast),Nearest Mipmap Anisotropic (Average),Linear Mipmap Anisotropic (Average)"), LIGHT_PROJECTOR_FILTER_LINEAR_MIPMAPS);
 
-	GLOBAL_DEF_RST("rendering/occlusion_culling/occlusion_rays_per_thread", 512);
+	GLOBAL_DEF_RST(PropertyInfo(Variant::INT, "rendering/occlusion_culling/occlusion_rays_per_thread", PROPERTY_HINT_RANGE, "1,2048,1,or_greater"), 512);
```

Comment:
Unsure if this should be `or_greater`

Reply:
I'm not experienced in engine internals. That's why I was unsure about what exact values should be an upper limits. Initially I added `or_greater` almost everywhere because assumed it safer then creating limit and then find that someone actually used larger value.
If you know exact upper limit values or could confirm here 2048 is good top value I will remove `or_greater`. Thanks!

Reply:
I have no clue, was just pointing out since it seems like a pretty performance critical thing given occlusion and threads

Reply:
Poke @clayjohn

Reply:
This value gets clamped internally between `viewport_size.x * viewport_size.y / (32 * 32)` and `viewport_size.x * viewport_size.y / (2 * 2)` after being multiplied by the number of threads. For a 1080x1920 screen with a CPU that supports 8 threads, the theoretical min is 254 and the theoretical max is 64,800.

For a 32 thread machine, it becomes 64 and 16,200. 

With a 4K monitor, the theoretical limits are much higher (with 8 threads, min is 1013, max is 259,200)

Accordingly, it makes sense to use `or_greater`